semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -145,12 +145,16 @@ object Javacp {
         new MethodSignatureVisitor
       )
       val typeDescriptor = {
-        val paramTypeDescriptors = signature.params.map {
+        val hasVarArg = method.access.hasFlag(o.ACC_VARARGS)
+        def toTypeDescriptor(t: JavaTypeSignature, i: Int): String = t match {
```

Comment:
had to make this recursive in order to print the `javaTypeSignature` of the Array type parameter.